translations/oe_corporate_blocks-sl.po

```diff
@@ -1,16 +1,132 @@
+msgid ""
+msgstr ""
 
-msgid "European Commission"
-msgstr "Evropska komisija"
```

Comment:
Did this string just got removed from the translations?

Reply:
Yes. it is fixed also for ET language.